scripts/file-finder.py

```diff
@@ -0,0 +1,52 @@
+import io
+import os
+from datetime import datetime
+import json
+from firecloud import fiss
+from firecloud.errors import FireCloudServerError
+import firecloud.api as fapi
+import numpy as np
+import pandas as pd
+
+# User-set variables
+BILLING_PROJECT_ID="biodata-catalyst"
+WORKSPACE="TSV-AFY"
+SUBDIRECTORY="\/case1\/"
+TABLE_NAME="CRAMs"
+# Make sure to escape the slashes in your SUBDIRECTORY variable. For instance, 
+# if your files are in a folder called "testfiles" you will need to enter it 
+# as "/\testfiles/\" or else Python will get angry.
+
+# Call FireCloud API
+try:
+    bucket = os.environ["WORKSPACE_BUCKET"]
+    response = fapi.list_entity_types(BILLING_PROJECT_ID, WORKSPACE)
+    if response.status_code != 200:
+        print("Error in Firecloud, check your billing project ID and the name of your workspace.")
+    else:
+        print("Firecloud has found your workspace!")
+        directory = bucket + SUBDIRECTORY
+except NameError:
+    print("Caught a NameError exception. This probably means you didn't restart the kernal after"
+          " running the first block of code (the one with all the imports). Run it again, restart"
+          " the kernal, then try running every block of code (including the import one) again.")
+
+# Append contents.txt with ls
```

Comment:
The mixed code might be confusing?  Also, generating the extra files can be avoided.  To do this in python rather than bash would be something like:

```
import google.cloud.storage

storage_client = google.cloud.storage.Client()

with open('final.tsv', 'w') as f:
    f.write(f"entity:{TABLE_NAME}_id\tfile_location\n")
    for file_path in return storage_client.list_blobs(bucket_name='wesout', prefix='TopMedAligner'):
        f.write(f'{file_path.split("/")[-1]}\t{file_path}\n')
```